packages/elements-core/src/components/Layout/SidebarLayout.tsx

```diff
@@ -31,22 +35,26 @@ export const SidebarLayout = React.forwardRef<HTMLDivElement, SidebarLayoutProps
           onMouseDown={(e: React.MouseEvent<HTMLElement>) => e.preventDefault()}
           style={{ maxWidth: `${SIDEBAR_MAX_WIDTH}px` }}
         >
-          <Flex
-            direction="col"
-            bg="canvas-100"
-            borderR
-            pt={8}
-            pos="sticky"
-            pinY
-            overflowY="auto"
-            style={{
-              paddingLeft: `calc((100% - ${maxContentWidth}px) / 2)`,
-              width: `${currentSidebarWidth}px`,
-              minWidth: `${SIDEBAR_MIN_WIDTH}px`,
-            }}
-          >
-            {sidebar}
-          </Flex>
+          {renderSideBar ? (
```

Comment:
can we not do renderSidebar && (...) rather than if/else

Reply:
Have become a fan of ternaries since reading this article https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

I think in this case we're safe to use && (the ternary remained from debugging efforts)

If using the ternary approach I personally prefer returning null instead of empty fragments.